### DISCUSSION TITLE
BetaCam, S-VHSV2, BetamaxV2

### DISCUSSION THREAD

**Discussion by Unit20** [ntsc-rs/ntsc-rs#193] [Presets]
Updated 2 presets with a new one called BetaCam.
[BetaCamSP.json](https://github.com/user-attachments/files/17177215/BetaCamSP.json)
[BetaMax.json](https://github.com/user-attachments/files/17177178/BetaMax.json)
[S-VHS-SP.json](https://github.com/user-attachments/files/17177179/S-VHS-SP.json)

![comparison again_000000](https://github.com/user-attachments/assets/66035bdd-604c-4fe6-832f-b3af06cfe4d7)

Some quick artwork for fun.

![T-NB_000000](https://github.com/user-attachments/assets/f8567f28-e9b6-4792-90e7-682b29dc4952)
![LV0N_000000](https://github.com/user-attachments/assets/dae89bae-740f-45e6-b89d-b0c9a69161d8)
![Checkerboard_000000](https://github.com/user-attachments/assets/ca2f1947-a83d-4999-ac3c-cb94029feb4e)

**Comment by notdigidani** [comment]
imma politely use this
